poderanan_michelle/activity3/Web Resume_Michelle Poderanan.html

```diff
@@ -0,0 +1,153 @@
+<DOCTYPE html>
+<html>
+
+<center>
+<br /><br />
+	<img src="Resume_Image.png" alt="This is a logo" height="160" width="140"/>
+	<h2>Michelle Del Socorro Poderanan</h2>
+	<h4>FullStack Web Developer</h4>
+	<h4>0977-007-9919 | <a href="michelle.poderanan@gmail.com">michelle.poderanan@gmail.com</a> | <a href="https://www.linkedin.com/in/michelle-poderanan-755099115/" target="_blank">https://www.linkedin.com/in/michelle-poderanan-755099115/</a> | <a href="https://www.facebook.com/chellemich06/" target="_blank">https://www.facebook.com/chellemich06/</a></h4>
+</center>	
+	<hr/>
+	
+	<body>
```

Comment:
Wrong use of body tag

`<!DOCTYPE html>
<html>
    <head></head>
   <body></body>
</html>
`